README.md

```diff
@@ -1,2 +1,93 @@
 # jslogger
 Logging Javascript logs to server.
+## Dependencies
+Jslogger need jquery to be imported.
+## Usage
+Download the src/jslogger-bundle.js and copy to desired location.
+Include jslogger.js in your page using below code.
+```javascript=
```

Comment:
@jaswanth098 @bajibabu9 I checked it in view mode and not seeing the javascript syntax highlighting.
It may be because of the extra `=` operator next to `javascript`.